.github/workflows/deployment.yaml

```diff
@@ -78,21 +78,35 @@ jobs:
         env:
           ARTIFACTORY_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }}
           ARTIFACTORY_PASSWORD: ${{ secrets.ARTIFACTORY_PASSWORD }}
-        run: curl --fail --user $ARTIFACTORY_USERNAME:$ARTIFACTORY_PASSWORD --upload-file dist/linux-64/jdk4py-*.tar.bz2 "$JDK4PY_CONDA_CHANNEL/linux-64/"
+        run: |
+          ls dist/linux-64/jdk4py-*.tar.bz2 | \
+          while read f; do
+            echo Uploading ${f}
+            curl --fail --user $ARTIFACTORY_USERNAME:$ARTIFACTORY_PASSWORD --upload-file ${f} "$JDK4PY_CONDA_CHANNEL/linux-64/"
+          done
 
       - name: Upload Conda package [macOS]
         if: matrix.os == 'macos-latest'
         env:
           ARTIFACTORY_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }}
           ARTIFACTORY_PASSWORD: ${{ secrets.ARTIFACTORY_PASSWORD }}
-        run: curl --fail --user $ARTIFACTORY_USERNAME:$ARTIFACTORY_PASSWORD --upload-file dist/osx-64/jdk4py-*.tar.bz2 "$JDK4PY_CONDA_CHANNEL/osx-64/"
+        run: |
+          ls dist/osx-64/jdk4py-*.tar.bz2 | \
+          while read f; do
+            echo Uploading ${f}
+            curl --fail --user $ARTIFACTORY_USERNAME:$ARTIFACTORY_PASSWORD --upload-file ${f} "$JDK4PY_CONDA_CHANNEL/osx-64/"
+          done
 
       - name: Upload Conda package [Windows]
         if: matrix.os == 'windows-latest'
         env:
           ARTIFACTORY_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }}
           ARTIFACTORY_PASSWORD: ${{ secrets.ARTIFACTORY_PASSWORD }}
-        run: curl --fail --user ${env:ARTIFACTORY_USERNAME}:${env:ARTIFACTORY_PASSWORD} "${env:JDK4PY_CONDA_CHANNEL}/win-64/" --upload-file "dist\win-64\jdk4py-${env:JDK4PY_VERSION}-0.tar.bz2"
+        run: |
+          curl --fail --user ${env:ARTIFACTORY_USERNAME}:${env:ARTIFACTORY_PASSWORD} "${env:JDK4PY_CONDA_CHANNEL}/win-64/" --upload-file "dist\win-64\jdk4py-${env:JDK4PY_VERSION}-py36_0.tar.bz2"
```

Comment:
We could follow [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html) and start support for Python >= 3.7:

```suggestion
```

Reply:
I will also change the `python_requires` to 3.7 in setup.py
This version has already been released with support for 3.6 but starting with next version it will only support Python >=3.7